src/Symfony/Bundle/TwigBundle/CacheWarmer/TemplateCacheCacheWarmer.php

```diff
@@ -27,14 +29,16 @@ class TemplateCacheCacheWarmer implements CacheWarmerInterface
 {
     protected $container;
     protected $finder;
+    protected $paths;
```

Comment:
yes please. Any new property should start as being private. We only switch to protected when we decide it should be a supported extension point (and I have no idea why we made the other properties protected btw, but changing it is a BC break)